### ISSUE TITLE
Update the Akka HTTP documentation page to reflect the current status in Play 2.5.x

### ISSUE THREAD

**Issue by TimMoore** [playframework/playframework#6917]
The first Google result I get when searching for "play akka-http" is https://www.playframework.com/documentation/2.5.x/AkkaHttpServer

This indicates that Akka HTTP support is experimental, but refers to Play 2.4 despite being in the 2.5.x docs: "In future versions of Play we may implement a production quality Akka HTTP backend, but in Play 2.4 the Akka HTTP server is mostly a proof of concept. We do not recommend that you use it for anything other than learning about Play or Akka HTTP server code. In Play 2.4 you should always use the default Netty-based server for production code."

@wsargent clarified to me that it is no longer considered experimental in 2.5.x (but is still disabled by default). It would be good to update the docs to say this.

### Play Version (2.5.x / etc)

2.5.x

### API (Scala / Java / Neither / Both)

Both

**Comment by micahkim23** [comment]
Can I work on this one?

**Comment by marcospereira** [comment]
Yes, @micahkim23. Thanks for contributing.

**Comment by micahkim23** [comment]
@marcospereira Since the library is no longer experimental, should I move it to under common topics? If it's no longer experimental in 2.5.x, is it also no longer experimental in 2.6.x?

**Comment by marcospereira** [comment]
@micahkim23 yes, you can move it to common topics.

You can submit a PR to master branch and, after it gets merged we can backport it to 2.5.x.

Again, thanks for taking care of this. :-)

**Comment by gmethvin** [comment]
We also need to change the artifact name if we want to make it non-experimental: https://github.com/playframework/playframework/blob/2.5.12/framework/project/Build.scala#L371

One of the reasons we marked the akka-http-server as experimental was that akka-http itself was marked as experimental. This means, among other things, that Akka can make breaking changes. So I agree we can declare it non-experimental for 2.5.x moving forward, but only because we've upgraded to a stable, non-experimental akka-http.
